rustls/tests/api.rs

```diff
@@ -5057,6 +5057,64 @@ fn test_client_rejects_illegal_tls13_ccs() {
     );
 }
 
+#[cfg(feature = "tls12")]
+fn remove_ems_request(msg: &mut Message) -> Altered {
```

Comment:
nit: I think this helper fn should be placed below the tests that use it.

Reply:
Moved